index.js

```diff
@@ -1,23 +1,21 @@
 'use strict';
+
+const copyProp = (to, from, prop) =>
+  Object.defineProperty(to, prop, Object.getOwnPropertyDescriptor(from, prop));
+
 module.exports = (to, from) => {
+	copyProp(to, from, 'length');
+
 	if (from.name) {
-		Object.defineProperty(to, 'name', {
-			value: from.name,
-			configurable: true
-		});
+		copyProp(to, from, 'name');
 	}
 
-	Object.defineProperty(to, 'length', {
-		value: from.length,
-		configurable: true
-	});
-
 	if (from.displayName) {
 		to.displayName = from.displayName;
 	}
```

Comment:
Hmm, actually, this is set by the user, so would already be handled by the loop below and since it's set by the user we don't have to check as it would probably never be an empty string unless the user set it to that.